### DISCUSSION TITLE
packageRules resets additionalBranchPrefix

### DISCUSSION THREAD

**Discussion by satanTime** [renovatebot/renovate#8087] [Request Help] [answered]
Looks like it's a bug.

```
  "semanticCommitScope": "{{#if baseDir}}{{baseDir}}{{else}}deps{{/if}}",
  "additionalBranchPrefix": "{{#if baseDir}}{{baseDir}}/{{else}}root/{{/if}}",
  "packageRules": [
  ]
```

that's what I have in the config here: https://github.com/satanTime/ng-mocks/blob/master/renovate.json#L10-L11

it worked correctly, all PR branches had `root/` prefix until I added packageRules here: https://github.com/satanTime/ng-mocks/blob/master/renovate.json#L12-L22

Now PRs for other dependencies don't respect `additionalBranchPrefix`, despite zero match in `packageRules`:
https://github.com/satanTime/ng-mocks/pull/346

As I understand it should be 2 PRs: one for `root/` and another one for `.circleci/`

**Comment by rarkins** [comment, marked as answer]
There's a significant difference between what you include here vs what is actually in your repo's config, so there may be something else going on. Please try to reproduce it in a minimal repo (minimal dependencies, minimal config).

**Reply by satanTime** [reply]
Yes, I removed repetitions, to simplify config for this issue.

**Reply by satanTime** [reply]
Let me know if it looks like a bug, I'll create an issue instead, and someone can investigate it deeper.

**Reply by rarkins** [reply]
Nobody will look into it unless there's a minimal reproduction repo, so you should create that first

**Reply by satanTime** [reply]
Here you go: [issue-repo.zip](https://github.com/renovatebot/renovate/files/5723504/issue-repo.zip)

**Reply by rarkins** [reply]
Can you create a public repo on github.com instead? i.e. demonstrating the problem

**Reply by satanTime** [reply]
Sorry, I don't get your point, I referred the github repo in the question. You asked to create a min repo, I did it. Now you ask github again, it's still in the question :)

**Reply by rarkins** [reply]
That repository is not a minimal reproduction. It has hundreds of files and 500+ lines of config. Isolate what you think the problem is to the smallest repository you can so that it is debuggable. e.g. minimum config and minimum dependencies

**Reply by satanTime** [reply]
Right, the config is the problem. If you had checked issue-repo.zip you would see that.

You could simply say that you don't want to investigate the issue instead of asking unrelated things.

I'll close the issue. Sorry for bothering you.
